app/views/new_user.erb

```diff
@@ -1,26 +1,39 @@
-<main>
+<div class="left-container"></div>
 
-  <div class="left-container"></div>
-  <form action="/user/new">
+<form action="/user/new">
+  <div class="form-group">
     <label for="username">Create Username</label>
     <input type="text" placeholder="Username" name="username"/>
+  </div>
 
+  <div class="form-group">
     <label for="password">Create Password</label>
     <input type="password" placeholder="Password" name="password"/>
+  </div>
 
+  <div class="form-group">
     <label for="password">Confirm Password</label>
     <input type="password" placeholder="Password" name="password"/>
+  </div>
 
+  <div class="form-group">
     <label for="first_name">First Name</label>
     <input type="text" placeholder="First Name" name="first_name"/>
+  </div>
 
+  <div class="form-group">
     <label for="last_name">Last Name</label>
     <input type="text" placeholder="Last Name" name="last_name"/>
+  </div>
 
+  <div class="form-group">
     <label for="email">Email</label>
     <input type="email" placeholder="Email" name="email"/>
+  </div>
 
+  <div class="form-group">
     <button type="submit">Submit</button>
-  </form>
-  <div class="right-container"></div>
-</main>
+  </div>
+</form>
+
+<div class="right-container"></div>
```

Comment:
Adding the `div.form-group`s puts the inputs onto separate lines, but it's also starting to set form up for Bootstrap